run/classifyCards.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+const groupBy = require('lodash.groupby');
+const mean = require('ml-array-mean');
+
+const { loadData, applyModel } = require('../src/svm');
+const filterMAD = require('../src/util/filterOutliersMAD');
+
+async function exec() {
+  const dirA = './data/esc-v2-all';
```

Comment:
can you "anonymise" this a bit? those dirs could be cli options instead of hardcoded